src/map/utils/battleutils.cpp

```diff
@@ -1825,6 +1829,10 @@ namespace battleutils
                 return base + skillModifier;
             }
         }
+        else if (PDefender->objtype == TYPE_TRUST && PDefender->GetMJob() == JOB_PLD)
+        {
+            base = 45;
```

Comment:
can we put a TODO here to verify if all PLD trusts have the same base block rate
They probably do, though.

Reply:
Sure can

Reply:
Added comment for follow up on retail block rate behaviors across the different trusts.